### PR TITLE
adjust highway context

### PR DESCRIPTION
**Pull request by wangkirin**
Signed-off-by: WangQilin <qilin.wang@huawei.com>

### CODE REVIEW

**Comment by tianxiaoliang** [comment]
rename "golang.org/x/net/context" to "context"

**Comment by wangkirin** [comment]
@tianxiaoliang  already changed~~

File: core/core_test.go [hunk 1]
```diff
@@ -26,7 +27,7 @@ func TestRPCInvoker_InvokeFailinChainInit(t *testing.T) {
 	config.GlobalDefinition = &model.GlobalCfg{}
 	invoker := core.NewRPCInvoker(core.ChainName(""))
 	replyOne := &helloworld.HelloReply{}
-	ctx := metadata.NewContext(context.Background(), map[string]string{
+	ctx := context.WithValue(context.Background(), highway.AttachmentsKey{}, map[string]string{
```
Comment (tianxiaoliang) [inline core/core_test.go]: highway.AttachmentsKey{} 
rest must not call highway package, move this method to util package

**Comment by wangkirin** [comment]
@tianxiaoliang already finished the changes requested

**Pull request merged**